lib/ohai/plugins/linux/network.rb

```diff
@@ -134,6 +138,25 @@ def parse_routes(family, iface)
     end.compact.flatten
   end
 
+  def ethernet_layer_one(iface)
```

Comment:
There's a lot of references to layer 1 in this PR.  Outside of fiber vs. twisted pair it's almost entirely at layer 2 on the OSI model.  You might want to update the specs and method names to be layer 2